neptune/new/attributes/atoms/copiable_atom.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2021, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import abc
+import typing
+
+from neptune.new.attributes.atoms.atom import Atom
+from neptune.new.internal.container_type import ContainerType
+from neptune.new.internal.operation import CopyAttribute
+from neptune.new.internal.utils.paths import parse_path
+from neptune.new.types.value_copy import ValueCopy
+
+if typing.TYPE_CHECKING:
+    from neptune.new.internal.backends.neptune_backend import NeptuneBackend
+
+
+class CopiableAtom(Atom):
```

Comment:
Shouldn't be `CopyableAtom`

Reply:
https://www.merriam-webster.com/ which I usually use to solve such dilemmas lists neither.

"Oxford languages by Google" matches copiable: https://www.google.com/search?q=copiable&oq=copiable , Collins -- copyable: https://www.collinsdictionary.com/dictionary/english/copyable . Wiktionary lists both: https://en.wiktionary.org/wiki/copiable

I like the visual look of Copiable better and considering the source material I think it's acceptable ;).